xml/cl.xml

```diff
@@ -6854,18 +6882,21 @@ server's OpenCL/api-docs repository.
             <require>
                 <type name="CL/cl.h"/>
             </require>
-            <require comment="cl_mem_properties">
+            <require>
+                <type name="cl_command_termination_reason_arm"/>
+            </require>
```

Comment:
I suspect this is a typo or merge error?  This type is not part of this extension.

```suggestion
```

Reply:
I think it should be part of the next extension (`cl_arm_controlled_kernel_termination`), rather than just deleted... I probably got tunnel vision and didn't see a separation between the 2 extensions when I did this.